src/lib/Components/Bidding/Context/__tests__/TimeOffsetProvider-tests.tsx

```diff
@@ -0,0 +1,65 @@
+import * as PropTypes from "prop-types"
+import React from "react"
+import { View } from "react-native"
+import * as renderer from "react-test-renderer"
+
+import { TimeOffsetProvider } from "../TimeOffsetProvider"
+
+jest.mock("../../../../metaphysics", () => ({ metaphysics: jest.fn() }))
+import { metaphysics } from "../../../../metaphysics"
+const mockphysics = metaphysics as jest.Mock<any>
+
+const SECONDS = 1000
+const MINUTES = 60 * SECONDS
+
+export class TestConsumer extends React.Component {
+  static contextTypes = {
+    timeOffsetInMilliSeconds: PropTypes.number,
+  }
+
+  render() {
+    return <View />
+  }
+}
+
+const dateNow = 1525983752000 // Thursday, May 10, 2018 8:22:32.000 PM UTC in milliseconds
+
+const consoleLog = console.log
+
+afterEach(() => {
+  console.log = consoleLog
+})
+
+it("injects timeOffsetInMilliSeconds as a context", () => {
+  // Debugging a timer is very tricky, so a few console.log calls are left in the code intentionally. Remove the line
```

Comment:
Ah, I guess this answers my question above.

Reply:
Yep, but I'm open to suggestions. Do you think that's a good thing? I'm also ok with just removing all the `console.log`s.

Reply:
I slightly updated this piece to make it self-documenting by wrapping with a `if (__DEV__ && typeof jest === "undefined") { ... }` guard.